src/tests/tribol_twb_integ.cpp

```diff
@@ -82,8 +64,8 @@ class TWBIntegTest : public ::testing::Test {
 
     for ( int a = 0; a < this->numNodes; ++a ) {
       for ( int ip = 0; ip < integ.numIPs; ++ip ) {
-        tribol::WachspressBasis( xy, integ.xy[dim * ip], integ.xy[dim * ip + 1], integ.xy[dim * ip + 2], this->numNodes,
-                                 a, phi );
+        tribol::WachspressBasis( xyz.data(), integ.xy[dim * ip], integ.xy[dim * ip + 1], integ.xy[dim * ip + 2],
```

Comment:
Same...I'm sure I'll see but did this function have to get refactored to accomodate the different coordinate layout?